modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -477,27 +400,14 @@ private CompletableFuture<Void> processTxRecoveryMessage(TxRecoveryMessage reque
 
         // Check whether a transaction has already been finished.
         if (txMeta != null && isFinalState(txMeta.txState())) {
-            return recoverFinishedTx(txId, txMeta)
-                    // If the sender has sent a recovery message, it failed to handle it on its own,
-                    // so sending cleanup to the sender for the transaction we know is finished.
-                    .whenComplete((v, ex) -> runCleanupOnNode(txId, senderId));
+            return runCleanupOnNode(txId, senderId);
```

Comment:
Am I correct that this doesn't actually change anything, just removes redundant code?